documentation/modules/auxiliary/admin/kerberos/ticket_converter.md

```diff
@@ -0,0 +1,50 @@
+# Kerberos Ticket Converting (CCache <---> Kirbi)
+
+The `ticket_converter` module is used to convert from a ccache file format to the kirbi file format and vice versa.
```

Comment:
Can we add additional detail here about the types of tickets there are maybe? And why you would want to convert between the two 👀

Reply:
ccache/kirbi; Potentially ticket dataformats is better lexicon to use? 🤔